app/views/businesses/check_duplicates.html.erb

```diff
@@ -0,0 +1,29 @@
+<%- model_class = Business -%>
+<div class="page-header">
+  <h1><%=t '.title', :default => model_class.model_name.human.pluralize.titleize %></h1>
+  <% if @business_import -%>
```

Comment:
This is not set in the controller action. Probably makes no sense either here.

Reply:
Resolved in a75fde9